src/sdam/state/mod.rs

```diff
@@ -372,17 +412,19 @@ impl TopologyState {
     /// Adds a new server to the cluster.
     ///
     /// A reference to the containing Topology is needed in order to start the monitoring task.
-    fn add_new_server(&mut self, address: StreamAddress, options: ClientOptions) {
+    fn add_new_server(
```

Comment:
Can you update the doc comment to indicate that a monitor is started as part of this method call to help prevent us from accidentally calling it in tests?

Reply:
I think the existing one conveys that, no?

>/// A reference to the containing Topology is needed in order to start the monitoring task.

What's funny is I did not add this comment, so it seems like this method used to start monitors but then it stopped doing so at one point.

Reply:
That's weird, I totally missed that during review. Maybe the github diff was hiding it.